packages/blocks/src/api/validation.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import Tokenizer from 'simple-html-tokenizer/dist/es6/tokenizer';
+import { Tokenizer } from 'simple-html-tokenizer';
```

Comment:
It was intentionally written the way it was previously in order to avoid pulling in [the very-large entities map](https://github.com/tildeio/simple-html-tokenizer/blob/master/src/generated/html5-named-char-refs.ts). On reflection, this wasn't very clear and could probably do for a comment. Is it not possible to keep this as it is and have just the `package.json` update?

Reply:
Oh I see, thanks for the update, I'll find a solution that avoids pulling this object 👍

Reply:
> Is it not possible to keep this as it is and have just the package.json update?